NebulaWorld/Chat/Commands/NavigateCommandHandler.cs

```diff
@@ -88,7 +100,7 @@ public string GetDescription()
         }
         public string GetUsage()
         {
-            return "[planet name | planet id | star name | star id | clear | player] [player id | playr name]";
+            return "[planet name | planet id | star name | star id | clear]\r\nUsage: /n player [player id | playr name]";
```

Comment:
I don’t think this solution is good. Instead change this method to return an array and change help command.